strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/event/artifact/MavenArtifactDownloadedEventListener.java

```diff
@@ -0,0 +1,181 @@
+package org.carlspring.strongbox.event.artifact;
```

Comment:
@fuss86  do we need t his class? Seems that it was mooved and I was not able to resolve this conflict.

Reply:
@sbespalov this class has been renamed to `MavenArtifactFetchedFromRemoteEventListener` and its full reference is `org.carlspring.strongbox.event.artifact.MavenArtifactFetchedFromRemoteEventListener`